apm-agent-plugins/apm-spring-webflux/apm-spring-webflux-testapp/pom.xml

```diff
@@ -96,7 +96,7 @@
                 <groupId>org.springframework.boot</groupId>
                 <artifactId>spring-boot-maven-plugin</artifactId>
                 <!-- explicit version required as bom will not cover plugins, only dependencies -->
-                <version>${version.spring-boot}</version>
```

Comment:
[for reviewer] the global `spring-boot` version property was removed because only used here in test application, thus in test code.